unidist/core/backends/mpi/core/communication.py

```diff
@@ -584,91 +621,6 @@ def recv_simple_operation(comm, source_rank):
     return comm.recv(source=source_rank)
 
 
-def send_operation_data(comm, operation_data, dest_rank, is_serialized=False):
```

Comment:
Why is this method removed?

Reply:
This function is no longer used. Before that, this function was used only by the send_operation function which is deleted bellow.